### PR TITLE
Update UML diagram for ModelClassDiagram

### LINKED ISSUE #121: Update UML diagram for ModelClassDiagram.puml

### PR DESCRIPTION
**Pull request by xqtann**
Update UML diagram (ModelClassDiagram) for Week 10 Task 5.
closes #121.

### CODE REVIEW

**Review by Potatoad88 — APPROVED** [review]
LGTM:)

File: docs/diagrams/ModelClassDiagram.puml [hunk 1]
```diff
@@ -19,6 +19,14 @@ Class Email
 Class Name
 Class Phone
 Class Tag
+Class BuyingProperties
+Class SellingProperties
+
+Class Property
```
Comment (OrangeCatLoves) [inline docs/diagrams/ModelClassDiagram.puml:25]: LGTM!

File: docs/diagrams/ModelClassDiagram.puml [hunk 2]
```diff
@@ -19,6 +19,14 @@ Class Email
...
+Class "<<enumeration>>\nHousingType" as HousingType
+Class PostalCode
+Class UnitNumber
+Class Price
```
Comment (OrangeCatLoves) [inline docs/diagrams/ModelClassDiagram.puml:29]: LGTM!

File: docs/diagrams/ModelClassDiagram.puml [hunk 3]
```diff
@@ -42,13 +50,27 @@ Person *--> Phone
...
+Person *--> SellingProperties
+
+SellingProperties --> "*" Property
+BuyingProperties --> "*" Property
+
+Property *--> HousingType
+Property *--> PostalCode
+Property *--> UnitNumber
+Property *--> Price
+Property *--> "*" Tag
```
Comment (OrangeCatLoves) [inline docs/diagrams/ModelClassDiagram.puml:63]: LGTM!

File: docs/diagrams/ModelClassDiagram.puml [hunk 4]
```diff
@@ -42,13 +50,27 @@ Person *--> Phone
...
+ModelManager -[hidden]right-> I
 
 Name -[hidden]right-> Phone
 Phone -[hidden]right-> Address
 Address -[hidden]right-> Email
+Email -[hidden]right-> BuyingProperties
+BuyingProperties -[hidden]right-> SellingProperties
 
-ModelManager --> "~* filtered" Person
+ModelManager -----> "~* filtered" Person
```
Comment (OrangeCatLoves) [inline docs/diagrams/ModelClassDiagram.puml:75]: LGTM!

**Review by OrangeCatLoves — APPROVED** [review]
LGTM!

**Pull request merged**